src/DependencyInjection/CmfRoutingAutoExtension.php

```diff
@@ -65,6 +65,12 @@ public function load(array $configs, ContainerBuilder $container)
                 $adapterName = 'doctrine_phpcr_odm';
             }
             $container->setParameter('cmf_routing_auto.persistence.phpcr.route_basepath', $config['persistence']['phpcr']['route_basepath']);
+
+            $loader->load('commands.xml');
+        }
+
+        if (isset($bundles['CmfRoutingAutoOrmBundle'])) {
+            $hasProvider = true;
```

Comment:
Is it not enough to set the adapter name to `doctrine_orm` in the config?

Reply:
It isn't. We have to add more config. You can see here https://github.com/wearemarketing/routing-auto-orm-bundle/blob/master/src/DependencyInjection/CmfRoutingAutoOrmExtension.php#L47

IMHO I leave all those configs to the new bundle. What do you think?

Reply:
If we are prepending the config in the ORM bundle, then shouldn't the adapter be configured here, and the exception should not be thrown?

Reply:
I am going to check it... because it makes sense what you say...